### DISCUSSION TITLE
Station logo if no metadata

### DISCUSSION THREAD

**Discussion by davinci1979** [Suplanus/Cuterdio#112] [Ideas]
Hello, on ios app if there is no metadata on the station link we always see only very nice app icon, instead of station logo. Can you fix it?
It is last thing which I cannot fix myself, beside that your app is perfect.

**Comment by Suplanus** [comment]
The station logos often in low resolution, and so it would be not so nice 🦄
With the Theme-Editor feature you can set a custom image.

**Comment by davinci1979** [comment]
> The station logos often in low resolution, and so it would be not so nice 🦄
> With the Theme-Editor feature you can set a custom image.

Yes, I bought Theme Editor, and its not working well with Carplay (sometime working, sometimes not), also I cannot customize each station (which can be perfect)

**Comment by Suplanus** [comment]
The custom logo is not set in notification center & Carplay... but it will come in future 🥳  
You can edit the station logo when you tap the `>` in the station list and change the image url:

![image](https://user-images.githubusercontent.com/2002916/90632160-3f7bcf00-e224-11ea-844e-422556884431.png)
(German UI)

**Comment by Suplanus** [comment]
This feature is available in 2.3.0 #140
